src/functions.js

```diff
@@ -103,16 +103,16 @@ function generateDockerFile(state) {
 
   //TODO bug! space at start
 
-  DockerFile += 'FROM bgruening/galaxy-stable:15.05\n\n';
-  DockerFile += 'MAINTAINER Björn A. Grüning, bjoern.gruening@gmail.com\n\n';
-  DockerFile += 'ENV GALAXY_CONFIG_BRAND "' + state.GALAXY_CONFIG_BRAND + '"\n\n';
-  DockerFile += 'WORKDIR /galaxy-central\n\n';
+  DockerFile += 'FROM ' + state.baseimage +'\n\r\n\r';
+  DockerFile += 'MAINTAINER Björn A. Grüning, bjoern.gruening@gmail.com\n\r\n\r';
+  DockerFile += 'ENV GALAXY_CONFIG_BRAND "' + state.GALAXY_CONFIG_BRAND + '"\n\r\n\r';
+  DockerFile += 'WORKDIR /galaxy-central\n\r\n\r';
```

Comment:
@jegalle are these line-breaks really needed? I don't think it's possible to build images on Windows.